nx-X11/programs/Xserver/GL/mesa/Imakefile.inc

```diff
@@ -1,9 +1,5 @@
 XCOMM $XFree86: xc/lib/GL/mesa/src/Imakefile.inc,v 1.7tsi Exp $
 
-#ifndef MesaBuildDir
-#define MesaBuildDir $(GLXLIBSRC)/mesa/
-#endif
-
 MESABUILDDIR = MesaBuildDir
```

Comment:
Hm, no, this won't work.

`MESABUILDDIR` will be empty, but seems to be used extensively in `nx-X11/lib/GL/mesa/main/Imakefile.inc`, which I guess is included by all mesa "subprojects"?

I'm not sure what to do about this, though, short of resurrecting `GLXLIBSRC`.